io.github.webcamoid.Webcamoid.yml

```diff
@@ -14,6 +14,7 @@ finish-args:
   - --filesystem=xdg-pictures
   - --filesystem=xdg-videos
   - --device=all
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
by default this is not allowed.

Reply:
This required for installing and detecting the virtual camera, is the [same rule used by OBS](https://github.com/obsproject/obs-studio/blob/e15a92e16fc3827027a956f6d6a1223b41aced61/CI/flatpak/com.obsproject.Studio.json#L17), which allow OBS to setup the virtual camera in it's [flatpak](https://flathub.org/apps/com.obsproject.Studio).